src/styles/App.module.css

```
@@ -0,0 +1,19 @@
+@import './Variable.module.css';
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  font-family: "Outfit", Arial, Helvetica, sans-serif;
+}
+
+html {
+  scroll-behavior: smooth;
+}
+
+.App {

```

Can we use `app` here for proper naming convention ?
```suggestion
.app {
```

---

fixed https://github.com/Sakshi-Simform/portfolio/pull/2/commits/806e9c821988805a9ab44516d8b41f942e61cdfb